src/usr/local/www/head.inc

```diff
@@ -461,8 +461,32 @@ if (($pagename === "index.php") && ($numColumns > 2)) {
 	}
 ?>
 		<ul class="context-links">
-<?php
 
+	<?php if ($dashboard_available_widgets_hidden): ?>
+		<li>
+			<a onclick="$('#widget-available').toggle(360);" title="<?=gettext("Show/Hide available widgets panel")?>">
+				<i class="btn-sm btn-breadcrumbbar">Widgets</i>
+			</a>
+		</li>
+	<?php endif?>
+
+	<?php if ($system_logs_filter_form_hidden): ?>
+		<li>
+			<a onclick="$('#filter-form').toggle(360)" title="<?=gettext("Show/Hide log filter panel")?>">
+				<i class="btn-sm btn-breadcrumbbar">Filter</i>
+			</a>
+		</li>
+	<?php endif ?>
+
+	<?php if ($system_logs_manage_log_form_hidden): ?>
+		<li>
+			<a onclick="$('#manage-log-form').toggle(360)" title="<?=gettext("Show/Hide manage log panel")?>">
+				<i class="btn-sm btn-breadcrumbbar">Manage Log</i>
+			</a>
```

Comment:
Agreed. There are a few places where it could be questionable, like "squid". Portuguese users are going to have a hard time trying to install "lula" :) I wondered if "Filter" falls into the same category.